packages/dashboard/src/features/analytics/components/posthog/RetentionCard.tsx

```
@@ -94,64 +69,78 @@ export function RetentionCard({ enabled }: { enabled: boolean }) {
+          </div>
+        ))}
       </div>
+
+      {/* data rows */}
+      {grid.map((row) => {
+        const size = row.cells[0]?.count ?? 0;
+        return (
+          <div
+            key={row.date}
+            className="flex items-center border-b border-[var(--alpha-8)] pl-1.5"
+          >
```

ESLint

[error] Replace `⏎············key={row.date}⏎············className="flex·items-center·border-b·border-[var(--alpha-8)]·pl-1.5"⏎··········` with `·key={row.date}·className="flex·items-center·border-b·border-[var(--alpha-8)]·pl-1.5"`

(prettier/prettier)